### PR TITLE
Added reset button to clear teams data, and added drafted positions under the team's name

### PR DESCRIPTION
**Pull request by amphro**
We might want to move the drafted positions based on real-estate, but I think it is good for now.

image

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -30,3 +30,19 @@ a, li a {
...
+.positions {
+  font-size: 70%;
+}
+
+.position {
+  padding: 2px;
+}
+
+.taken {
+  text-decoration: line-through;
```
Comment (amphro) [inline public/stylesheets/style.css]: I was just looking at the screenshot again. I should have added color: green in .position, and color: red for .taken. We can add that later.

File: public/javascripts/nfl-data.js [hunk 2]
```diff
@@ -90,17 +90,27 @@ function NFLData() {
...
-      "dataType": "json"
-    })
-    .done(function (data) {
-      if (position == 'DEF') {
-        getTeams(data);
-      } else {
-        getPlayers(data);
-      }
-    });
+    if (NFLData.USE_CACHED_PLAYER_DATA) {
```
Comment (amphro) [inline public/javascripts/nfl-data.js:93]: Setting a flag to determine whether to use the cached json files or not. I didn't want to have to keep adding this back in.

File: views/layout.jade [hunk 3]
```diff
@@ -20,6 +20,7 @@ html
     script(src='https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js')
     script(src='/javascripts/bootstrap.js')
     script(src='/javascripts/nfl-data.js')
+    script NFLData.USE_CACHED_PLAYER_DATA = #{USE_CACHED_PLAYER_DATA || false};
```
Comment (amphro) [inline views/layout.jade:23]: Not sure if this is the best way to set the flag in client JS from an env variable. Let me know if you want me to change it.

File: public/javascripts/draftee.js [hunk 4]
```diff
@@ -40,7 +46,13 @@ $(function() {
...
       $('#playerInfo').text('Defense');
     } else {
-      $('#playerInfo').text(draftable.position + ' - #' + draftable.number + ' - ' + draftable.team);
+      var info = $('#playerInfo');
+      var position = draftable.position;
+      var playerNumber = draftable.number;
+      var team = draftable.team;
+
+      info.data('position', position);
+      $('#playerInfo').text(position + ' - #' + playerNumber + ' - ' + team);
```
Comment (justinappler) [inline public/javascripts/draftee.js]: info.text since you already have the jQuery object?

File: views/layout.jade (hunk 3 above)
Comment (justinappler) [inline views/layout.jade:23]: I think down the road the best way would be to have the server get the player data, and serve it up to the client. Then only the server needs to know whether the nfl data is cached or not. This is good for now though.

**Pull request merged**